testing/MLDBFB-509_pushed_non_printable_char_cant_query.py

```diff
@@ -0,0 +1,43 @@
+#
+# MLDBFB-509_pushed_non_printable_char_cant_query.py
+# Mich, 2016-05-02
+# This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
+#
+import unittest
+
+mldb = mldb_wrapper.wrap(mldb)  # noqa
+
+class PushNonPrintableCharCantReadTest(MldbUnitTest):  # noqa
+
+    @unittest.expectedFailure
+    def test_unprintable_row_name(self):
+        ds = mldb.create_dataset({'id' : 'ds1', 'type' : 'sparse.mutable'})
+        barbarous_name = 'coco' + chr(17)
+        ds.record_row(barbarous_name, [['colA', 1, 0]])
+        ds.commit()
+
+        # crash!
```

Comment:
If MLDB crashes would that make the unit test to fail?  I think so and therefore we should skip the test not marked them as `expectedFailure`.